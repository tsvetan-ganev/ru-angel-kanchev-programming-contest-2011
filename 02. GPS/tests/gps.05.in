257

Ajtos
Aksakovo
Alfatar
Antonovo
Apriltsi
Ardino
Asenovgrad
Aheloj
Ahtopol
Balchik
Bankja
Bansko
Banja
Batak
Batanovtsi
Belene
Belitsa
Belovo
Belogradchik
Beloslav
Berkovitsa
Blagoevgrad
Bobov dol
Boboshevo
Bojurishte
Bojchinovtsi
Boljarovo
Borovo
Botevgrad
Bratsigovo
Bregovo
Breznik
Brezovo
Brusartsi
Burgas
Buhovo
Balgarovo
Bjala (Oblast Varna)
Bjala (Oblast Ruse)
Bjala Slatina
Bjala cherkva
Varna
Veliki Preslav
Veliko Tarnovo
Velingrad
Vetovo
Vetren
Vidin
Vratsa
Valchedram
Valchi dol
Varbitsa
Varshets
Gabrovo
General Toshevo
Glavinitsa
Glodjevo
Godech
Gorna Orjahovitsa
Gotse Delchev
Gramada
Guljantsi
Gurkovo
Galabovo
Dve mogili
Debelets
Devin
Devnja
Djebel
Dimitrovgrad
Dimovo
Dobrinishte
Dobrich
Dolni chiflik
Dolna banja
Dolna Mitropolija
Dolna Orjahovitsa
Dolni Dabnik
Dospat
Dragoman
Drjanovo
Dulovo
Dunavtsi
Dupnitsa
Dalgopol
Elena
Elin Pelin
Elhovo
Etropole
Zavet
Zemen
Zlataritsa
Zlatitsa
Zlatograd
Ivajlovgrad
Ignatievo
Iskar
Isperih
Ihtiman
Kableshkovo
Kavarna
Kazanlak
Kalofer
Kamen
Kameno
Kaolinovo
Karlovo
Karnobat
Kaspichan
Kermen
Kilifarevo
Kiten
Klisura
Kneja
Kozloduj
Kojnare
Koprivshtitsa
Kostandovo
Kostenets
Kostinbrod
Kotel
Kocherinovo
Kresna
Krivodol
Krichim
Krumovgrad
Kubrat
Kuklen
Kula
Kardjali
Kjustendil
Levski
Letnitsa
Lovech
Loznitsa
Lom
Lukovit
Laki
Ljubimets
Ljaskovets
Madan
Madjarovo
Malko Tarnovo
Marten
Mezdra
Melnik
Merichleri
Mizija
Momin prohod
Momchilgrad
Montana
Maglij
Nedelino
Nesebar
Nikolaevo
Nikopol
Nova Zagora
Novi Iskar
Novi pazar
Obzor
Omurtag
Opaka
Orjahovo
Pavel banja
Pavlikeni
Pazardjik
Panagjurishte
Pernik
Perushtitsa
Petrich
Peshtera
Pirdop
Plachkovtsi
Pleven
Pliska
Plovdiv
Polski Trambesh
Pomorie
Popovo
Pordim
Pravets
Primorsko
Provadija
Parvomaj
Radnevo
Radomir
Razgrad
Razlog
Rakitovo
Rakovski
Rila
Roman
Rudozem
Ruse
Sadovo
Samokov
Sandanski
Sapareva banja
Sveti Vlas
Svilengrad
Svishtov
Svoge
Sevlievo
Senovo
Septemvri
Silistra
Simeonovgrad
Simitli
Slavjanovo
Sliven
Slivnitsa
Slivo pole
Smoljan
Smjadovo
Sozopol
Sopot
Sofija
Sredets
Stambolijski
Stara Zagora
Strajitsa
Straldja
Strelcha
Suvorovo
Sungurlare
Suhindol
Saedinenie
Sarnitsa
Tvarditsa
Tervel
Teteven
Topolovgrad
Trojan
Tran
Trastenik
Trjavna
Tutrakan
Targovishte
Ugarchin
Hadjidimovo
Harmanli
Haskovo
Hisarja
Carevo
Car Kalojan
Chepelare
Cherven brjag
Chernomorets
Chiprovtsi
Chirpan
Shabla
Shivachevo
Shipka
Shumen
Jablanitsa
Jakoruda
Jambol
