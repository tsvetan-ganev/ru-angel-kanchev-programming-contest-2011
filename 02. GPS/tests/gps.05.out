a b v g d e z i k l m n o p r s t u h c j